### DISCUSSION TITLE
Can I save my blobs if my IOS is not signed for my device?

### DISCUSSION THREAD

**Discussion by MasterMarius-zz** [airsquared/blobsaver#332] [Q&A] [answered]
Hy, I have an Iphone SE (gen 1) that is now on IOS 14.0, but didn't save any blobs for this version.
I know all the data neccesary for it to be generated (Hex ECID, Device ID, Board Config, Generator and AP Nonce "name" eg: d88e9a56b80.... ), but don't kow if it's possible to generate it now.
Can I stiil generate it even it is not signer the IOS on witch I am now?
Please if you have an answer mail me at master.marius@gmail.com

**Comment by airsquared** [comment, marked as answer]
You can only save blobs for currently signed versions.

**Reply by cohabitable** [reply]
Out of curiosity, would you be interested in a pull request to allow it to save blobs for older devices, e.g. something like this [A5-A6X-Blobs-Dumper](https://github.com/RossDarker/A5-A6X-Blobs-Dumper) process?
